app/main.py

```diff
@@ -2,0 +1,29 @@
+import os
+
+from contextlib import contextmanager
+
+
+@contextmanager
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[0] == "mv" and len(command_list) == 3:
+        mv_dir = command_list[-1].split("/")
+        with open(command_list[1], "a"):
+            pass
+        if len(mv_dir) == 1:
+            with open(command_list[1],
+                      "r") as file_in, open(mv_dir[-1],
+                                            "w") as file_out:
+                file_out.write(file_in.read())
+            os.remove(command_list[1])
+        else:
+            full_path = ""
+            for f_dir in mv_dir[:-1]:
+                full_path += f_dir + "/"
```

Comment:
The code currently concatenates paths using string operations. It is recommended to use `os.path.join` for path concatenation to ensure cross-platform compatibility.